test/integration/webhook.js

```diff
@@ -274,23 +268,26 @@ describe('Webhooks', function () {
 		expect(event).to.not.be.null;
 		expect(event.channel_type).to.eq(chan.type);
 		expect(event.channel_id).to.eq(chan.id);
-		expect(event.members[0].user).to.be.an('object');
-		expect(event.members[0].user.online).to.eq(true);
-		expect(event.members[0].user.unread_count).to.eq(1);
-		expect(event.members[0].user.channel_unread_count).to.eq(1);
-		expect(event.members[0].user.total_unread_count).to.eq(1);
-		expect(event.members[0].user.unread_channels).to.eq(1);
-		expect(event.members[0].user.channel_last_read_at).to.not.be.null;
-
-		expect(event.members[1].user).to.be.an('object');
-		expect(event.members[1].user.online).to.eq(false);
-		expect(event.members[1].user.unread_count).to.eq(0);
-		expect(event.members[1].user.channel_unread_count).to.eq(0);
-		expect(event.members[1].user.total_unread_count).to.eq(0);
-		expect(event.members[1].user.unread_channels).to.eq(0);
-		expect(event.members[1].user.channel_last_read_at).to.not.be.null;
-		const lastRead = new Date(event.members[0].user.channel_last_read_at);
+		expect(event.members).to.have.lengthOf(3);
```

Comment:
is 3 correct?

Reply:
Yes, should be thierry, tommaso and horatiu